docker-compose.yaml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
+    app:
+        image: gentux/php:lumen-5.6-ci
+        volumes:
+            - ./:/var/www
+        ports:
+            - 81:80
```

Comment:
should we set this back to 80:80?

Reply:
no i left it as 81 as anytime you're going to spin it up inorder to develop locally it will need to be on a different port than the app you'll be testing it in. Our apps almost always run on 80.